olive/evaluator/olive_evaluator.py

```diff
@@ -1007,11 +1007,11 @@ def _evaluate_raw_latency(
         return results["latencies"]["total_inference_time"][warmup_num:]
 
     def _prepare_dataloader(
-        self, dataloader: Dataset, model: SNPEModelHandler, file_list_batch_size=None
+        self, dataloader: Dataset, model: SNPEModelHandler, file_chunk_size=None
     ) -> FileListDataLoader:
         if isinstance(dataloader, FileListDataLoader):
             return dataloader
-        return FileListCommonDataLoader(dataloader, model.io_config, batch_size=file_list_batch_size)
+        return FileListCommonDataLoader(dataloader, model.io_config, batch_size=file_chunk_size)
```

Comment:
do we need rename the parameter batch_size of FileListCommonDataLoader to file_chunk_size?

Reply:
Maybe not. There are still many other cases which create the file list data loader directly. Let us say user prepare the local inputs text(like what we did for mobilenet/inception/vgg case), they still need the concept of batch_size to run batch inputs.

Reply:
In mobilenet/inception/vgg case, the batch_size is used as normal batch_size like dataset? I see the source code https://github.com/microsoft/Olive/blob/main/olive/platform_sdk/qualcomm/utils/data_loader.py#L73 use the batch size as chunk. No?

Reply:
For the file list data loader, the batch size always performs like `file chunk`.

I am thinking for Mobilenet case, when user create the the local file and file list data loader, then they begin to iterate the data loader. Then `batch_size` may looks better? As it is a common concept in any data loader.

Reply:
If you insist to choose the `file_chunk_size`, maybe I can do it in a separated PR? I do not know how many example configs need to be updated.

Reply:
does the parameter changes need to update the config json?

Reply:
I am just thinking the batch_size is a common term associated with dataloader. If we use same term but for different purpose, we'd better use a different name.